examples/examples.js

```diff
@@ -35,36 +36,37 @@ import Themes from './themes';
  * Here you can add any extra examples with the Card label as the key, and the component to render as the value
  */
 export default {
-  'Array Value Columns': ArrayValueColumns,
```

Comment:
Let's undo these diffs. It's good to have indent in an object, and I don't want this sort of thing to go back and forth in various PRs, so the way it was before was fine.

Reply:
> As far as the actual fix for the scroll behavior, this turns out to be a little more complicated than we thought. Implementing the auto overflow behavior removes the ability to apply a fixed header (for y axis). So that's a bit of a problem, as it would be a breaking change. I experimented around a bit, and I found that if we apply the style
> 
> ```
> position: aboslute;
> ```
> 
> to the `MUIDataTable-paper` class at the top level `div` for the whole library component, it fixes the overflowing behavior problem, and allows fixed header options. However, it's also a different type of display from `scrollFullHeight`. So what I think we should do is keep `scrollFullHeight` as is, as this will still be a viable option for tables that don't have many columns, and add a new option (also to maintain backwards compatibility) called `scrollFullHeightFullWidth` which implements the `position: absolute` as the fix.

position:absolute seems to collide when you have the Material-UI Grid around the table.
respsonsive setting set to 'scrollFullHeightFullWidthÄ
````
<Grid container justify="center">
        <Grid item xs={12} md={10}>
          <MUIDataTable title={<Title />} data={rawData} columns={TableColumns} options={TableSettings}></MUIDataTable>
        </Grid>
      </Grid>```
screenshot with the position: absolute
<img width="988" alt="20-02-06-congstar VPP Order List-bq1fo" src="https://user-images.githubusercontent.com/622841/73924825-5e8cc200-48cd-11ea-983b-02a316b75de3.png">

<img width="1029" alt="table-with-pos-abs" src="https://user-images.githubusercontent.com/622841/73924896-72d0bf00-48cd-11ea-99f7-581dd112130f.png">



screenshot without ```.MUIDataTable-paperResponsiveScrollFullHeightFullWidth-106 {
    position: absolute;
}```
<img width="988" alt="table-without-pos-abs" src="https://user-images.githubusercontent.com/622841/73924905-75331900-48cd-11ea-907d-ffda3edb87df.png">